concepts/05 Widgets/Floating Action Button/00 Getting Started with Floating Action Button/00 Getting Started with Floating Action Button.md

```diff
@@ -12,7 +12,7 @@ From this tutorial, you will learn how to create a single- or multi-action FAB a
 
```

Comment:
Line 3: `The FAB can perform an action or open a stack of two to five related actions (speed dial)`
I'd like to rewrite this without using "perform". Can we say "execute" instead?
Also, the period is missing in this sentence.

Line 5: `To find more details on the FAB concept and best practices, refer to the [Material Design Guidelines](https://material.io/design/components/buttons-floating-action-button.html).`
Can we update this to the following?

`For more details on the FAB concept and best practices, refer to the following topic: [Material Design Guidelines](https://material.io/design/components/buttons-floating-action-button.html).`